boost-maven/boost-maven-plugin/src/it/test-jaxrs-2.0/pom.xml

```diff
@@ -31,22 +31,20 @@
 
 	<dependencyManagement>
 		<dependencies>
-			<dependency>
-				<groupId>io.openliberty.boost</groupId>
-				<artifactId>boost-bom</artifactId>
-				<version>@pom.version@</version>
-				<scope>import</scope>
-				<type>pom</type>
-			</dependency>
+		<dependency>
+			<groupId>io.openliberty.boosters</groupId>
```

Comment:
There was a comment in an internal call or Gitter that this "groupId" should be "io.openliberty.boost.boosters" instead.   Personally, since I think this isn't the last input we'll hear on this, I think it's OK to merge either way, as long as we highlight this as an open question in ongoing design reviews.